Modules/Bridge/VTK/include/itkVTKImageExport.h

```diff
@@ -128,14 +128,14 @@ class ITK_TEMPLATE_EXPORT VTKImageExport : public VTKImageExportBase
   BufferPointerCallback() override;
 
 private:
-  std::string m_ScalarTypeName;
-  int         m_WholeExtent[6];
-  int         m_DataExtent[6];
-  double      m_DataSpacing[3];
-  double      m_DataOrigin[3];
-  double      m_DataDirection[9];
-  float       m_FloatDataSpacing[3];
-  float       m_FloatDataOrigin[3];
+  std::string m_ScalarTypeName{};
```

Comment:
I guess this is not needed for STL types?

Reply:
> I guess this is not needed for STL types?

Strictly speaking, initialization by `{}` is not needed for STL containers, but it doesn't harm either. It just default-constructs them.